fastlane/lib/fastlane/actions/artifactory.rb

```diff
@@ -10,7 +10,12 @@ def self.run(params)
         Actions.verify_gem!('artifactory')
 
         require 'artifactory'
+
+        UI.user_error!("Cannot connect to Artifactory - 'username' was provided but it's missing 'password'") if params[:username] && !params[:password]
+        UI.user_error!("Cannot connect to Artifactory - 'password' was provided but it's missing 'username'") if !params[:username] && params[:password]
+        UI.user_error!("Cannot connect to Artifactory - either 'api_key', or 'username' and 'password' must be provided") if !params[:api_key] && !params[:username]
         file_path = File.absolute_path(params[:file])
+        
```

Comment:
Can you run `bundle exec rubocop -a` again? I think this is causing CI to fail (trailing whitespace infraction)